extension.toml

```diff
@@ -1,6 +1,6 @@
 id = "kotlin"
 name = "Kotlin"
-version = "0.0.2"
+version = "0.0.3"
```

Comment:
Let’s leave off the version bump in this PR.

We can bump the version when we go to publish.

Generally it’s better to separate functional changes from bumping the version.

Reply:
> But I can see that all three of them comprise one pull request; you're saying to split it into two separate pull requests then?

Correct.

That way we can merge your functional change but then make additional changes before a v0.0.3 release (if needed/desired).